docs/src/index.md

```diff
@@ -109,11 +109,12 @@ TVNeg
 ```
 
 Consistent with common notation, transforms are applied right-to-left; for example, `as(Real, ∞, 3)` is equivalent to `TVShift(3) ∘ TVNeg() ∘ TVExp()`.
+If you are working in an editor where typing Unitful is difficult, `TransformVariables.compose` (not exported) is provided as an alias for `∘`, as in `TransformVariables.compose(TVScale(5.0), TVExp(), TVNeg())`.
```

Comment:
perhaos "typing Unitful" was meant to be "typing Unicode"?

Reply:
yep